<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" dir="ltr" lang="en">
<head>
	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
	<meta name="keywords" content="Fatal error in ProductionOrderSFCInBOD with ION 10x/Tools 10.5, baan,baanerp,erp,forum,discussion,bulletin board" />
	<meta name="description" content="[Archive] Fatal error in ProductionOrderSFCInBOD with ION 10x/Tools 10.5 ION" />
	
	<title>Fatal error in ProductionOrderSFCInBOD with ION 10x/Tools 10.5 [Archive]  - Baanboard.com</title>
	<link rel="stylesheet" type="text/css" href="../styles.css" />
</head>
<body>
<div class="pagebody">
<div id="navbar"><a href="index.html">Newbaanboard</a> &gt; <a href="index.html">Baan SIGs</a> &gt; <a href="index.html">ION</a> &gt; Fatal error in ProductionOrderSFCInBOD with ION 10x/Tools 10.5</div>
<hr />
<div class="pda"></div>

<hr />

<div class="post"><div class="posttop"><div class="username">alex.bonutti</div><div class="date">6th July 2017, 13:06</div></div><div class="posttext">Dears,<br />
after we updated our LN to Tools 10.5, we are receiving from ION a fatal error like below on ProductionOrderSFCInBOD:<br />
<br />
FATAL ERROR: bfiter.skip.initials: iterator 0 not found Can not continue in otlbctbodp in DLL: ottstpamdll (dal.do.update.dependent.fields)[bfiter.skip.initials]<br />
Stack trace (package combination '61oa9ing'):<br />
 dal.do.update.dependent.fields(-2,0)  (in object ottstpamdll)<br />
 dal.save.object(&quot;tisfc001&quot;)  (in object ottstpamdll)<br />
 reporting.completed.production.order(&quot;PRD041074&quot;,1,1)  (in object otiboddll1111)  tibod.dll1111.hook.afterexecute.productionordersfcinbod.change(&quot;PRD041074&quot;,1,1,0,0,&quot;&quot;,<br />
<br />
<br />
This is limited to one company only, where the backflush is used and set as automatic.<br />
Infor has no answers for almost 2 months now. Did anyone ever encounter or handle this kind of error?<br />
<br />
Best regards,<br />
Alex</div></div><hr />


<div class="post"><div class="posttop"><div class="username">Zakir Hussain</div><div class="date">20th December 2018, 08:26</div></div><div class="posttext">Hello Guys, even i got this error while inserting new records in the main table using dal.new.object() through a dll. I also had dependencies defined in the before.open.object()<br />
After hours of debug it was noticed that i was querying on the main table in the before.new.object() of the tables DAL, and based on a suggestion from a colleague when i commented the query on main table and relocate the piece of code in another function, the error stopped coming.<br />
<br />
you can try to see if this is the case in your scenario.<br />
<br />
regards<br />
zakir</div></div><hr />



</div>
</body>
</html>